### PR TITLE
Updates CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by jinxcat**
Update CONTRIBUTORS.md

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -44,7 +44,7 @@ Steps:
 2. Create a ``feature`` branch to hold your development changes:
 
    ```bash
-   $ git checkout -b my-feature
+   $ git checkout -b {feature|issue}/feature-name
```
Comment (TanjaCrijns) [inline CONTRIBUTING.md]: Wasn't it going to be feature/issue/feature-name?

Comment (jinxcat) [reply CONTRIBUTING.md]: yes

**Comment by anneschuth** [comment]
This is a duplicate of #14 it seems.

**Comment by jinxcat** [comment]
@anneschuth partly. Will rebase to clean.

**Comment by jinxcat** [comment]
@anneschuth  @TanjaCrijns now?

**Pull request merged**